notebooks/image_models/solutions/object_detection_automl_vertex.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
<p>Maybe we can add this command to check the actual csv file</p><p> </p><pre class="ql-syntax" spellcheck="false">!head ../dataset_import_files/salads.csv
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>